salvo/src/lib/test_source_tree.py

```diff
@@ -9,83 +9,240 @@
 from src.lib import (cmd_exec, source_tree)
 import api.source_pb2 as proto_source
 
+logging.basicConfig(level=logging.DEBUG)
 log = logging.getLogger(__name__)
 
 def test_source_tree_object():
   """Verify that we throw an exception if not all required data is present."""
-  pass
+  st = source_tree.SourceTree(proto_source.SourceRepository())
```

Comment:
The Python style guide advises to use descriptive names, even if they are slightly longer. Can we use a more descriptive identifier instead of just `st`? Consider `source` or `tree`.

Applies to multiple locations below.

Reply:
Fixed